src/contentscript/overlay/root/components/DappletImage/index.tsx

```diff
@@ -1,9 +1,8 @@
-import { initBGFunctions } from 'chrome-extension-message-wrapper'
 import cn from 'classnames'
-import React, { FC, useEffect, useRef, useState } from 'react'
-import { browser } from 'webextension-polyfill-ts'
+import React, { FC } from 'react'
 import { StorageRef } from '../../../../../background/registries/registry'
```

Comment:
Looks that the `StorageRef` type is used as a shared type between `background` and `contentscript`. The shared types are located in `/src/common/types.ts` in this repo.